website/docs/reference/warehouse-profiles/bigquery-profile.md

```diff
@@ -6,16 +6,16 @@ title: "BigQuery Profile"
 
 BigQuery targets can be specified using one of three methods:
 
-1. oauth
+1. oauth via `gcloud`
+2. oauth via access token(s)
 2. a service account file
 3. service account json
 
 For local development, we recommend using the oauth method. If you're scheduling dbt on a server, you should use the service account auth method instead.
 
 BigQuery targets should be set up using the following configuration in your `profiles.yml` file.
 
-### oauth Authentication
-
+### OAuth Local Authentication
```

Comment:
Can we make these headings match the above list? It's a little easier to navigate when they are consistent